frontend/src/pages/cert-manager/CertificatesPage/components/certificateUtils.ts

```diff
@@ -0,0 +1,88 @@
+import { CertSubjectAlternativeNameType } from "@app/pages/cert-manager/PoliciesPage/components/CertificateTemplatesV2Tab/shared/certificate-constants";
+
+export enum FrontendSanType {
+  DNS = "dns",
+  IP = "ip",
+  EMAIL = "email",
+  URI = "uri"
+}
+
+export const mapBackendSanTypeToFrontend = (backendType: string): FrontendSanType => {
+  switch (backendType) {
+    case CertSubjectAlternativeNameType.DNS_NAME:
+      return FrontendSanType.DNS;
+    case CertSubjectAlternativeNameType.IP_ADDRESS:
+      return FrontendSanType.IP;
+    case CertSubjectAlternativeNameType.EMAIL:
+      return FrontendSanType.EMAIL;
+    case CertSubjectAlternativeNameType.URI:
+      return FrontendSanType.URI;
+    default:
+      return backendType as FrontendSanType;
+  }
+};
```

Comment:
why do we need this in the first place? why doesn't the enums just translate directly?